tools/sigclient/functionalQueries/search_with_eval.json

```diff
@@ -2,7 +2,7 @@
     "queryText": "http_method=POST AND (app_name=Bracecould OR app_name=R*) | regex app_name=\".*could\" | eval abs_sum=abs(latitude) | eval concated=abs_sum.latitude.\"_\".ltrim(gender, \"fem\") | eval round_sum=round(latitude, 2) | eval avg_sum_min_max= (latitude + longitude + latitude)/3 | eval cnt=latency%123 | eval len_gender=len(gender) | eval request_status=if(http_status >= 400, \"Error\", \"Success\") | eval bool_success=if(in(http_status, 400, 404, 500), \"0\", 1) | eval is_string=if(isstr(bool_success), bool_success, null()) | eval decoded_url=urldecode(\"http%3A%2F%2Fwww.splunk.com%2Fdownload%3Fr%3Dheader\") | where request_status in(\"Success\") | eval splitVal=split(concated, \".\") | eval max_cnt=max(162, cnt) | eval exact_per=exact(cnt/max_cnt) | eval gender_short=if(match(gender, \"^m\"), \"A\", \"F\") | eval toNumber=tonumber(bool_success) | eval status_hex=tostring(http_status, \"hex\") | eval substr_res=substr(\"splendid\", 1, 3) . substr(\"chunk\", -3) | eval logValue=log(http_status, 2) | eval formatted_time=strftime(1719415380000, \"%Y\") | eval timeType=typeof(formatted_time) | eval short_gen=if(searchmatch(\"gender=m*e\"), \"M\", \"F\") | eval x=\"hi\" | eval y=\"a\" | eval z=\"great\" | eval match=if(searchmatch(\"x=hi y=a? z=*\"), \"yes\", \"no\") | eval nowResult=relative_time(now(),\"-1h\") | eval timeResult=relative_time(time()/1000,\"-1h\") | eval finalResult=if(nowResult=timeResult, \"Pass\", \"Fail\") | fields abs_sum, app_name, avg_sum_min_max, bool_success, cnt, concated, decoded_url, exact_per, finalResult, formatted_time, gender, gender_short, http_method, http_status, ident, is_string, len_gender, logValue, longitude, ltrim, match, max_cnt, request_status, round_sum, short_gen, splitVal, status_hex, substr_res, timeType, toNumber, x, y, z",
     "expectedResult": {
         "totalMatched": {
-            "value": 13,
+            "value": 7,
```

Comment:
Why is this getting changed? Was the old pipeline giving the wrong result?

Reply:
I think this was a typo earlier, the correct answer is 7 so just updated it.